packages/server/lib/controllers/xhrs.coffee

```diff
@@ -79,10 +87,13 @@ module.exports = {
       {data: bytes, encoding: encoding}
 
   getResponse: (resp, config) ->
+    if resp.then
+      return resp
```

Comment:
reshape the resolved object with the `data` payload and remove it from the other area